packages/neo-one-smart-contract-lib/src/ownership/Ownable.ts

```diff
@@ -0,0 +1,62 @@
+import { Address, SmartContract, createEventNotifier } from '@neo-one/smart-contract';
+
+const ownershipTransferred = createEventNotifier<Address | undefined, Address>('transferred_ownership', 'from', 'to');
+
+export function Ownable<TBase extends Constructor<SmartContract>>(base: TBase) {
+  abstract class OwnableClass extends base {
+    protected abstract initialOwner: Address;
+    private mutableOwner: Address | undefined = undefined;
+    private mutableOwnerInitialized = false;
+
+    // tslint:disable-next-line:no-any
+    public constructor(...args: any[]) {
```

Comment:
Is there a reason the constructor was added back in?

Reply:
Yes, testing apparatus for something I was trying, but it was left behind.  thnx.